### ISSUE TITLE
detach() in Encoder

### ISSUE THREAD

**Issue by FIRE-MOS** [eriklindernoren/PyTorch-GAN#142]
loss_c_1 = lambda_cont * criterion_recon(c_code_12, c_code_1.detach())
loss_c_2 = lambda_cont * criterion_recon(c_code_21, c_code_2.detach())
Could someone tell me why use detach() here??
